R/utility.R

```diff
@@ -179,3 +179,28 @@ getJobResult <- function(jobId = "", ...){
 
   return(results)
 }
+
+validateClusterConfig <- function(clusterFilePath){
+  if (file.exists(clusterFilePath)) {
+    pool <- rjson::fromJSON(file = clusterFilePath)
+  }
+  else{
+    pool <- rjson::fromJSON(file = file.path(getwd(), clusterFilePath))
+  }
+
+  stopifnot(!is.null(pool$poolSize))
```

Comment:
Random question - is there a 'stopif' method? You can avoid doing the double negative since this reads funny:

stopif(is.null(pool$poolSize))